contrib/go-pg/pg.v10/pg_go.go

```diff
@@ -17,12 +17,22 @@ import (
 )
 
 // Wrap augments the given DB with tracing.
-func Wrap(db *pg.DB) {
+func Wrap(db *pg.DB, opts ...ClientOption) {
+	cfg := new(clientConfig)
+	defaults(cfg)
+	for _, opt := range opts {
+		opt(cfg)
+	}
+	h := &queryHook{
+		cfg: cfg,
+	}
 	log.Debug("contrib/go-pg/pg.v10: Wrapping Database")
-	db.AddQueryHook(&queryHook{})
+	db.AddQueryHook(h)
```

Comment:
I don't think we need to declare `h`. 
`db.AddQueryHook(&queryHook{cfg: cfg})` is clear enough.

Reply:
fixed, thanks